core/system/src/main.c

```diff
@@ -50,6 +51,11 @@ UVISOR_NOINLINE void uvisor_init_post(void)
         /* init MPU */
         vmpu_init_post();
 
+        /* init page allocator */
```

Comment:
You sure? The style is completely mixed.
I'm also not in favor of going over all my source code and changing this manually now.
A python script with some regex can do this for the entire uVisor source at once.